parsers/meta-m4.h

```diff
@@ -28,8 +28,8 @@ struct m4ParserClient
 	langType lang;
 	void *data;
 
-	char quoteOpen;
-	char quoteClose;
+	const char quoteOpen;
+	const char quoteClose;
```

Comment:
`changequote()` is handled in `m4Quote{Open|Close}` in m4.c.
The const'ed fields are for initializing `m4Quote{Open|Close}`.